File: Easy/Length_of_last_word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int t=s.length();
        int c=0;
        for(int i=t-1;i>=0;i--)
        {
            if(s[i]==' '&&c!=0)
                return c;
             else if((c==0)&&(s[i]==' '))
                 continue;
            else
                 c++;
        }
            return c;
         }
};

File: lengthoflastword.py
class Solution:
    def lengthOfLastWord(self, s):
        end = len(s) - 1
        while end > 0 and s[end] == " ": end -= 1
        beg = end
        while beg >= 0 and s[beg] != " ": beg -= 1
        return end - beg
